receiver/hostmetricsreceiver/internal/scraper/cpuscraper/metadata.yaml

```diff
@@ -30,3 +30,10 @@ metrics:
     gauge:
       value_type: double
     attributes: [cpu, state]
+
+  system.cpu.number:
```

Comment:
This probably should be called `system.cpu.count`. See https://github.com/open-telemetry/semantic-conventions/blob/4456e4d1d4c2858ffd495a5cc6dc02f802dbaec4/specification/metrics/semantic_conventions/README.md#use-count-instead-of-pluralization